### DISCUSSION TITLE
Is there any way to replace sorting icpns in primeng table component?

### DISCUSSION THREAD

**Discussion by cloudy-repo** [primefaces/community#1862] [PrimeNG] [answered]
this line creates sorting icons on table header:

`<p-sortIcon field="field" ></p-sortIcon>`

 and the icons are always 
```
pi-sort-alt
pi-sort-amount-down-alt
pi-sort-amount-up-alt
```

![image](https://github.com/primefaces/community/assets/92725158/380ce4e9-a0a8-43a6-82ca-59efc871f850)

is there any way to replace these icons with other primg ng icons

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi, you can replace the existing icons with your desired icon using the "sorticon" template. I have created an example for you.  https://stackblitz.com/edit/pzkjxs-vkku4c?file=src%2Fapp%2Ftable-removable-sort-demo.html

**Reply by Hatem-UrWave** [reply]
This is angualr 17 right ? 
I tried it in angular 19 and it didnt work as expected , it keeps the default icons and then add my icons beside it 
how can i hide the default icons in angular 19 ?

**Comment by mh-zwave** [comment]
I have the same problem!
